frontend-react/public/locales/de/translation_de.json

```diff
@@ -1167,12 +1167,366 @@
+    "ink consumption": "Tintenverbrauch",
+    "Ink tank recovery from empty to full (squid form)": "Tintentank Regenerationszeit (Tintenfischform)",
+    "frames": "Bilder",
+    "seconds": "Sekunden",
+    "Run speed": "Laufgeschwindigkeit",
+    "Run speed (firing)": "Laufgeschwindigkeit (beim Schießen)",
+    "distance units / frame": "Distanzeinheiten / Bild",
+    "Swim speed": "Schwimmgeschwindigkeit",
+    "Points to special": "Punkte zur Spezialwaffe",
+    "speed": "Geschwindigkeit",
+    "pointsShort": "p",
+    "Special lost when killed": "Spezialleistenverlust beim Tod",
+    "Special lost when killed mid-Splashdown": "Spezialleistenverlust beim Tod während eines Tintenschocks",
+    "of the charge": "der Spezialleiste",
+    "duration": "Dauer",
+    "Special Power Up also increases Inkjet's shots' painting and blast radius": "Spezialwaffe+ erweitert auch den Explosions- und Färbradius des Tintendüsers",
+    "reticle size": "Fadenkreuzgröße",
+    "ink coverage": "Färbeffizienz",
+    "180dmg hitbox size": "180-Schaden Trefferradius",
+    "70dmg hitbox size": "70-Schaden Trefferradius",
+    "55dmg hitbox can't be increased with Special Power Up so the total radius of the special doesn't change": "der 55-Schaden Trefferradius sowie der maximalradius können durch Spezialwaffe+ nicht erweitert werden",
+    "activation time": "Aktivierungszeit",
+    "Amount inked by Ink Storm is not increased only in how long distance the droplets are spread. Special Power Up also increases the distance you can throw the seed.": "Die Größe der gefärbten Fläche eines Tintenschauers ist invariabel und ändert sich nur in der Verteilung, Spezialwaffe+ erhöht zudem die Wurfdistanz.",
+    "durability": "Dauer",
+    "55dmg explosion hitbox size": "55-Schaden Trefferradius",
+    "bubble size": "Blasengröße",
+    "explosion hitbox": "Explosionsradius",
+    "autocharge speed": "Autoladegeschwindigkeit",
+    "time": "Zeit",
+    "Quick Respawn activates when enemy kills you twice without you getting a kill in between": "Schnelle Rückkehr aktiviert sich nur wenn du ohne einen Gegner zu eliminieren eliminiert wirst",
+    "(in the air)": "(in der Luft)",
+    "(on the ground)": "(auf dem Boden)",
+    "range and velocity": "Reichweite und Geschwindigkeit",
+    "full-power phase duration": "Dauer der Hochleistungsphase",
+    "mid-phase duration": "Dauer der Mittelleistungsphase",
+    "tracking duration": "Markierdauer",
+    "tracking range": "Aufspürreichweite",
+    "boost": "Bonus",
+    "When jumping to Sub Power Up boosted beakons QSJ AP bonus is applied on top of any existing QSJ the jumper has. 57AP can't be exceeded. Value shown is the bonus if the user of Beakon has 0AP invested in QSJ.": "Wenn man mit Sekundärstärke+ verstärkte Sprungbojen benutzt, werden dem Springer je 3 EP Supersprung+ hinzugefügt. 57 EP kann nicht überschritten werden. Der angezeigte Wert entspricht dem EP Zusatz..",
```

Comment:
"When jumping to Sub Power Up boosted beakons QSJ AP bonus is applied on top of any existing QSJ the jumper has. 57AP can't be exceeded. Value shown is the bonus if the user of Beakon has 0AP invested in QSJ.": "Wenn man mit Sekundärstärke+ verstärkte Sprungbojen benutzt, werden dem Springer Supersprung+ Bonus-EP hinzugefügt. 57 EP kann nicht überschritten werden. Der angezeigte Wert entspricht dem EP-Zusatz.",